src/app/progress/progress-list/progress-list.component.html

```diff
@@ -0,0 +1,70 @@
+
+  <clr-datagrid>
+    <clr-dg-column [clrDgField]="'progress'" class="color">Progress</clr-dg-column>
```

Comment:
I think `[clrDgField]="'progress'"` can be removed. This only makes the field filterable by the progress attribute, however as this is displayed as a bar it does not need to be filtered